### PR TITLE
[Doc] add readme of tutorials

### PR DESCRIPTION
**Pull request by kevinintel**
## Type of Change

Documentation

## Description

add readme of tutorials

## Expected Behavior & Potential Risk

n/a

## How has this PR been tested?

n/a

## Dependency Change?

n/a

### CODE REVIEW

File: docs/tutorials/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Tutorials
+===========================
+Feel free to delve into these tutorials and unlock the features in just a matter of minutes. 
+
+
+| Feature |task |Model| Notebook Link        |
```
Comment (hshen14) [inline docs/tutorials/README.md:6]: Task

File: docs/tutorials/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+| ------- | --------------------------------------------- | ----------------- | -------------------- |
```
Comment (hshen14) [inline docs/tutorials/README.md:7]: why some models n/a?

File: docs/tutorials/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+|Static and dynamic post training quantization|Token Classification|Distilbert Base|[Notebook](./pytorch/token-classification/distilbert_base_ner.ipynb)|
+|Dynamic post training quantization|Translation|T5 small|[Notebook](./pytorch/translation/t5-small.ipynb)|
+|Early Exit (SWEET)|Text Classification|Bert Base|[Notebook](./pytorch/text-classification/SWEET.ipynb)|
+|Early Exit (TangoBERT)|Text Classification|Roberta Base + Bert Tiny|[Notebook](./pytorch/text-classification/SWEET.ipynb)|
+|SetFit|Text Classification|MiniLM|[Notebook](./pytorch/text-classification/SetFit_model_compression_AGNews.ipynb)|
+|Static and dynamic post training quantization|Text Classification|Bert Base|[Notebook](./pytorch/text-classification/bert-base-uncased-MRPC.ipynb)|
+|Distillation|Text Classification|n/a|[Notebook](./pytorch/text-classification/distillation.ipynb)|
+|Pruning|Text Classification|Bert base|[Notebook](./pytorch/text-classification/pruning.ipynb)|
+|Prune Once for All|Text Classification|Bert base|[Notebook](./pytorch/text-classification/orchestrate_optimizations.ipynb)|
```
Comment (hshen14) [inline docs/tutorials/README.md:25]: Shall we order by name? Prune Once for All appears multiple times, but not in order

File: docs/tutorials/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+|Static and dynamic post training quantization|Question Answering|MiniLM (Distilled from RoBERTa Large)|[Notebook](./pytorch/question-answering/Dynamic_MiniLM_SQuAD.ipynb)|
+|Static and dynamic post training quantization|Question Answering|Bert Large|[Notebook](./pytorch/question-answering/bert-large-uncased-whole-word-masking-finetuned-squad.ipynb)|
+|Dynamic post training quantization|Summarization|Bert Base|[Notebook](./pytorch/summarization/pegasus-samsum.ipynb)|
+|Static and dynamic post training quantization|Token Classification|Distilbert Base|[Notebook](./pytorch/token-classification/distilbert_base_ner.ipynb)|
+|Dynamic post training quantization|Translation|T5 small|[Notebook](./pytorch/translation/t5-small.ipynb)|
+|Static and dynamic post training quantization|Text Classification|Bert Base|[Notebook](./pytorch/text-classification/bert-base-uncased-MRPC.ipynb)|
+|Pruning|Text Classification|Bert base|[Notebook](./pytorch/text-classification/pruning.ipynb)|
+|Distillation|Question Answering|enter the model name|[Notebook](./pytorch/question-answering/distillation.ipynb)|
```
Comment (hshen14) [inline docs/tutorials/README.md:17]: enter the model name - what does it mean?

Comment (kevinintel) [reply docs/tutorials/README.md:17]: user need to input model name or path

Comment (hshen14) [reply docs/tutorials/README.md:17]: Specified by user

**Pull request merged**